recipes-app/node-red/node-red-package.inc

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) Siemens AG, 2022
+#
+# Authors:
+#  Felix Moessbauer <felix.moessbauer@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+# common setting for all node-red-contrib modules
+DEBIAN_DEPENDS .= ", node-red"
+RDEPENDS += "node-red"
+# install directory of nod-red-* modules
+NPM_LOCAL_INSTALL_DIR = "/root/.node-red"
```

Comment:
Maybe rather model as class? Would allow to move the common `inherit npm` over as well.

Reply:
Moved to #397

Reply:
Modeled as class.